src/python/pants_release/generate_github_workflows.py

```diff
@@ -858,16 +840,17 @@ def build_wheels_job(
                     "env": helper.platform_env(),
                 },
                 helper.upload_log_artifacts(name="wheels-and-pex"),
-                *([deploy_to_s3("Deploy wheels to S3")] if for_deploy_ref else []),
                 *(
                     [
                         {
-                            "name": "Upload Pants PEX",
+                            "name": "Upload Wheels and Pex",
                             "if": "needs.release_info.outputs.is-release == 'true'",
                             # NB: We can't use `gh` or even `./pants run 3rdparty/tools/gh` reliably
                             #   in this job. Certain variations run on docker images without `gh`,
                             #   and we could be building on a tag that doesn't have the `pants run <gh>`
                             #   support. `curl` is a good lowest-common-denominator way to upload the assets.
+                            # NB: Each job will re-upload the same testutil wheel. It's easier than
```

Comment:
Are we sure this is robust? Will concurrent uploads succeed?

Why isn't it straightforward to have just the linux job do it?

Reply:
I think I can get it working, leveraging the generator.